packages/sanity/src/desk/panes/document/documentPanel/header/DocumentHeaderTitle.tsx

```diff
@@ -21,12 +23,24 @@ export function DocumentHeaderTitle(): ReactElement {
   }
 
   if (!documentValue) {
-    return <>New {schemaType?.title || schemaType?.name}</>
+    return (
+      <>
+        {t('header.document-header-title.new.text', {item: schemaType?.title || schemaType?.name})}
+      </>
+    )
   }
 
   if (error) {
-    return <>Error: {error.message}</>
+    return <>{t('header.document-header-title.error.text', {error: error.message})}</>
```

Comment:
Wondering how to reason about these errors that will most likely be in english. Fine for now, but should we flag these somehow for later review?

Reply:
Maybe we can add a `todo` here. Maybe something like `// @todo: i18n review second pass`?